Majasdarbs/03-majasdarbs/CV/CSS/normalize.css

```diff
@@ -0,0 +1,27 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    zoom: 1;
```

Comment:
Why do you need zoom here?
This is also non-standard style: https://developer.mozilla.org/en-US/docs/Web/CSS/zoom